src/main/java/core/basesyntax/CustomFileReader.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+import java.util.List;
+
+public interface CustomFileReader {
+    public List<String> read(String fileName);
```

Comment:
According to the Java conventions, the 'public' modifier is redundant in interface method declarations, as all methods in an interface are implicitly public. You can remove the 'public' keyword here for cleaner code.